src/utils/api.js

```diff
@@ -19,4 +19,12 @@ export const getSingleArticle = (article_id) => {
         return response.data.article;
     });
    
+}
+
+export const patchVote = (article_id, value) => {
+    return articleApi
+    .patch(`/articles/${article_id}`, { inc_votes: value})
+    .then((response) => {
+        return response.data.article;
```

Comment:
Looking at this it seems fine, but for some reason the votes weren't persisting for me after a refresh of the page. So I don't think it's actually patching successfully, either that or the frontend is only displaying `0` and not the actual number of votes an article has